docs/node-operators/migration-guides/upgrade-to-0710.md

```diff
@@ -0,0 +1,401 @@
+
+There is no change required in the `vega core` config. If you have to prepare the `vega` config from scratch, use the instruction for [v0.68.2](./upgrade-to-0682.md#7-update-vega-core-config).
+
+### 8. Update Tendermint config
+
+There is no change required in the `tendermint` config. If you have to prepare the `tendermint` config from scratch, use the instruction for [v0.68.2](./upgrade-to-0682.md#8-update-tendermint-config).
+
+### 9. Migrate tendermint data when using default home path
+
+Vega has migrated from the tendermint to its fork, called the [CometBFT](https://cometbft.com). This change has been introduced in the previous version of the vega. If you are using the default home path for the Tendermint(`~/.tendermint`), we recommend migrating it to the default for the CometBFT(`~/.cometbft`). You can read more about the change and required steps in the [previous migration guide](./upgrade-to-0702.md#9-migrate-tendermint-data-when-using-default-home-path)
+
+### 10. Update data node config
+
+#### The Gateway Port settings
+
+We have merged the `GraphQL` and the `REST` APIs into a single port. Now you must define the `Port` and `IP` in the `[Gateway]` section instead of its child. Remove the `Port` and `IP` from the `[Gateway.GraphQL]` and `[Gateway.REST]` sections.
+
+```diff
+  [Gateway]
++   Port = 443
++   IP = "0.0.0.0"
+    [Gateway.GraphQL]
+-     Port = 3008
+-     IP = "0.0.0.0" 
+      ...
+    [Gateway.REST]
+-     Port = 3008
+-     IP = "0.0.0.0" 
+      ...
+```
+
+#### Enable the TLS on the data node API
+
+The data node can request the TLS certificate for you automatically. The only requirement is to use port 443 for the Gateway component and has this port open to the public internet. You can still use a custom port and some software like Apache or Nginx and proxy pass requests to the data node, but in this case, you are responsible for generating and renewing the certificate.
+
+Below is an example certificate for the data node auto TLS:
+
+```toml
+[Gateway]
+  Port = 443
+  IP = "0.0.0.0"
+  Level = "Info"
+  Timeout = "5s"
+  SubscriptionRetries = 3
+  GraphQLPlaygroundEnabled = true
+  MaxSubscriptionPerClient = 250
+  HTTPSEnabled = true
+  AutoCertDomain = "api.vega.example.com"
+  CertificateFile = ""
+  KeyFile = ""
+  [Gateway.GraphQL]
+    Enabled = true
+    ComplexityLimit = 0
+    Endpoint = "/graphql"
+  [Gateway.REST]
+    Enabled = true
+    APMEnabled = true
+  [Gateway.CORS]
+    AllowedOrigins = ["*"]
+    MaxAge = 7200
+...
+```
+
+If the `api.vega.example.com` domain points to your data node, it will obtain the certificate from the Let's Encrypt.
+
+:::note All the differences
+To see all the differences, we recommend generating a new configuration and compare with your configuration on the node. The procedure is described [here](./upgrade-node.md#16-update-data-node-config)
+:::
+
+### 11. Update the vegavisor configuration
+
+---
+title: Upgrade to 0.71.0
+sidebar_label: Upgrade to 0.71.0
+---
+
```

Comment:
What's going on here, onwards? This is formatting for a new page - should it be a separate page or is this like a duplicate?